website_and_docs/content/documentation/webdriver/elements/locators.en.md

```diff
@@ -370,12 +370,36 @@ others it's as simple as setting a parameter in the FindElement function
 
 ### ByChained
 
-The `ByChained` class enables you to _chain_ two By locators together. For example, instead of having to locate a parent element, and then a child element of that parent, you can instead combine those two `FindElement` functions into one.
+The `ByChained` class enables you to _chain_ two By locators together. For example, instead of having to locate a parent element, and then a child element of that parent, you can instead combine those two `FindElement()` functions into one.
```

Comment:
HI @shbenzer ,
Could you please limit the line length to 100 characters as per the style guidelines

See Line length in https://www.selenium.dev/documentation/about/style/

Reply:
Ah, thought I fixed that but you're right that second line is over 100. My bad

Reply:
Thank you!